MODULE.bazel

```diff
@@ -0,0 +1,332 @@
+module(name = "buildbuddy")
+
+bazel_dep(name = "bazel_skylib", version = "1.5.0")
+bazel_dep(name = "platforms", version = "0.0.8")
+bazel_dep(name = "rules_proto", version = "5.3.0-21.7")
+
+# Needed for com_google_protobuf
+bazel_dep(name = "rules_python", version = "0.31.0")
+
+python = use_extension("@rules_python//python/extensions:python.bzl", "python")
+python.toolchain(
+    configure_coverage_tool = False,
+    # Do not error if root user is detected
+    ignore_root_user_error = True,
+    python_version = "3.11",
+)
+
+bazel_dep(name = "zlib", version = "1.3")
+bazel_dep(name = "bazel_features", version = "1.4.1", repo_name = "io_bazel_rules_go_bazel_features")
+
+version = use_extension("@io_bazel_rules_go_bazel_features//private:extensions.bzl", "version_extension")
+use_repo(version, "bazel_features_globals", "bazel_features_version")
+
+bazel_dep(name = "rules_go", version = "0.46.0", repo_name = "io_bazel_rules_go")
+bazel_dep(name = "gazelle", version = "0.35.0", repo_name = "bazel_gazelle")
+archive_override(
+    module_name = "gazelle",
+    integrity = "sha256-MpOL2hbmcABjA1R5Bj2dJMYO2o15/Uc5Vj9Q0zHLMgk=",
+    patch_strip = 1,
+    patches = [
+        "@@//buildpatches:gazelle.patch",
+    ],
+    urls = [
+        "https://mirror.bazel.build/github.com/bazelbuild/bazel-gazelle/releases/download/v0.35.0/bazel-gazelle-v0.35.0.tar.gz",
+        "https://github.com/bazelbuild/bazel-gazelle/releases/download/v0.35.0/bazel-gazelle-v0.35.0.tar.gz",
+    ],
+)
+
+go_sdk = use_extension("@io_bazel_rules_go//go:extensions.bzl", "go_sdk")
+go_sdk.download(
```

Comment:
```suggestion
# TODO: Remove explicit `goarch`/`goos` after the next rules_go update:
# https://github.com/bazelbuild/rules_go/commit/145b042e21a073fa3637c096bf9779cdbcac6f75
go_sdk.download(
```

Reply:
Nice that it was fixed. I was hoping that I could get to it once I merged this PR 👀 

Will add a note next round